Add MySQL App Service / ACA - fix typos and styles

john0isaac opened the pull request:
## Purpose
<!-- Describe the intention of the changes being proposed. What problem does it solve or functionality does it add? -->
* Add Flask MySQL XX App Service
* Add Flask MySQL XX ACA
* fix small typos and styles

After building this I used Cookiecutter to generate the repo I previously created and there were no changes in the code.
- https://github.com/john0isaac/azure-flask-mysql-flexible-appservice
- https://github.com/john0isaac/azure-flask-mysql-flexible-aca

## Does this introduce a breaking change?
<!-- Mark one with an "x". -->
```
[X] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

john0isaac committed:
add mysql to infra

john0isaac committed:
Add mysql-flexible to conftests

john0isaac committed:
Add mysql to abbreviation

john0isaac committed:
rename without change to be more generic

john0isaac committed:
add mysql docker image

john0isaac committed:
rename without change to be more generic

john0isaac committed:
add mysql to pages macros

john0isaac committed:
add mysql to cookiecutter and postgen config

john0isaac committed:
add mysql to readme and fix some typos

john0isaac committed:
Add mysql devcontainer config and partially fix #47

john0isaac committed:
add test to mysql and ignore md changes for other tests

john0isaac committed:
renamed without change to be more generic

john0isaac committed:
add mysql to flaskapp

john0isaac committed:
change port according to seleciton

john0isaac committed:
add reference to readme

john0isaac commented:
Commits after this comment are related to ACA setup as it's built on top of the mysql setup

john0isaac committed:
add mysql ACA

john0isaac commented on templates/conftest_flask_sql.py:
yes, http://dev.mysql.com/doc/refman/5.0/en/gone-away.html

kjaymiller commented on {{cookiecutter.__src_folder_name}}/.github/workflows/test_postgres.yml:
```diff
@@ -37,8 +46,8 @@ jobs:
           python-version: ${{ matrix.python_version }}
-          architecture: x64
           {% endraw %}
+          architecture: x64
```
Is this going to be a problem when testing against ARM?

john0isaac replied:
This change is not needed but I asked myself why are we disabling interpretation for `architecture: x64`
it's not related to jinja syntax so, it's safe to do it like that and the wrapping was not needed.

kjaymiller commented on README.md:
```diff
@@ -81,6 +82,8 @@ To request support please create a [new discussion](https://github.com/kjaymille
 - [Flask Azure Container Apps Postgres Addon  Azure Container Apps](https://github.com/Azure-Samples/azure-flask-postgres-addon-aca)
 - [Flask CosmosDB - MongoDB Adapter  Azure Container Apps](https://github.com/Azure-Samples/azure-flask-cosmos-mongodb-aca)
 - [Flask CosmosDB - MongoDB Adapter  Azure App Service](https://github.com/Azure-Samples/azure-flask-cosmos-mongodb-appservice)
+- [Flask MySQL - Flexible Server Azure App Service](https://github.com/john0isaac/azure-flask-mysql-flexible-appservice)
```
I think this is a temporary good choice to include working samples. In the long term. There should be samples that are created in the Azure-Samples Repo

john0isaac committed:
update mysql bicep to be consistent

john0isaac committed:
fix mysql connection error for django

error: django.db.utils.OperationalError: (2002, Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2))

john0isaac committed:
fix production env vars for django

Django has the same env vars for development and production. My production env vars started with Azure_something which made django deployment fail

john0isaac committed:
fix password typo

john0isaac committed:
Merge pull request #2 from john0isaac/add-django-mysql

fix mysql django password typo

john0isaac committed:
fix env var to match local name

john0isaac committed:
fix dbname to be consistent
